packages/eas-cli/src/build/configure.ts

```diff
@@ -96,5 +92,40 @@ export async function ensureEasJsonExistsAsync(ctx: ConfigureContext): Promise<v
 
   await fs.writeFile(easJsonPath, `${JSON.stringify(easJson, null, 2)}\n`);
   await gitAddAsync(easJsonPath, { intentToAdd: true });
-  log.withTick('Created eas.json file');
+  log.withTick('Successfully generated eas.json.');
+}
+
+enum ShouldCommitChanges {
+  Yes,
+  ShowDiffFirst,
+  Skip,
+}
+
+async function reviewAndCommitChangesAsync(
+  commitMessage: string,
+  askedFirstTime: boolean = true
+): Promise<void> {
+  const { selected } = await promptAsync({
+    type: 'select',
+    name: 'selected',
+    message: 'Can we commit these changes to git for you?',
+    choices: [
+      { title: 'Yes', value: ShouldCommitChanges.Yes },
+      ...(askedFirstTime
+        ? [{ title: 'Show the diff and ask me again', value: ShouldCommitChanges.ShowDiffFirst }]
+        : []),
+      {
+        title: 'Skip committing changes, I will do it later on my own',
+        value: ShouldCommitChanges.Skip,
+      },
+    ],
+  });
+
+  if (selected === ShouldCommitChanges.Yes) {
+    await commitPromptAsync(commitMessage);
+    log.withTick('Successfully committed changes.');
```

Comment:
```suggestion
    log.withTick('Committed changes');
```
Same with others, super simple logging.

Reply:
I was using messages that @brentvatne prepared on notion (linked in PR description). 

cc @brentvatne